### ISSUE TITLE
Create block in tabular template, to facilitate external lib

### ISSUE THREAD

**Issue by shinneider** [sehmaschine/django-grappelli#956]
Hi, i am a developer of `[django-admin-inline-paginator](https://github.com/shinneider/django-admin-inline-paginator) lib

and i have two issue for incompatibility to this lib. 
I will act on these problems to solve this.

but if you can create a block in the tabular template for me, it will facilitate and guarantee compatibility for future changes in your lib's.
i need block in `templates/admin/edit_inline/tabular.html` in end of first div. 
see this [PR](https://github.com/shinneider/django-admin-inline-paginator/issues/4)

is possible to create ?
and make sense for this lib ?

Thanks

**Comment by sehmaschine** [comment]
You mean a block, not an include, right? Because I could not find a block with the referenced issue. But yes, including a generic block with tabular (and/or stacked) is possible.

**Comment by shinneider** [comment]
@sehmaschine yes, it's a block.
this is not a error in your lib. It's a improvement to help-me in my lib...

because, without block i need to copy all your templade and override, but with block i just extend and add my logic in block.
provavelmente guarantee compatibility for future changes in your template...

**Comment by sehmaschine** [comment]
Sure sure, I get it. Please tell me where exactly (refer to the line!) you want to have that block. At best, please provide a PR.

**Comment by shinneider** [comment]
@sehmaschine 
Sorry for the long time to response.

PR Created.
Thanks

**Comment by sehmaschine** [comment]
@shinneider thanks! unfortunately, the PR is hard to follow (at least when checking via GitHub). did you just add the two blocks (stacked_content, tabular_content) or did you do anything else?

**Comment by shinneider** [comment]
@sehmaschine i just added the two blocks and moved actual code to inside...
